website/docs/learn-graphql/guides/the-design-of-graphql.md

```diff
@@ -90,9 +80,9 @@ Because inherently, our rendering logic for `ConversationInfo` and its data-spec
 
 So if we want to do this, if we want each component to be able to specify its own data needs, how can we do so? The realization is that our data-specification has a key property that it needs to fulfill, which is composition.
 
-## Composition
+### Composition
 
-Composition in GraphQL is achieved by leveraging fragments, which are snippets of a query that can be composed together to form larger queries. These fragments are colocated with their components and composed into a tree that very much follows the shape of the component tree.
+Composition in GraphQL is achieved by leveraging fragments, which are snippets of a query that can be composed together to form larger queries. These fragments are co-located with their components and composed into a tree that very much follows the shape of the component tree.
```

Comment:
I think a little more explanation here could help the reader key in on the important parts of the following code samples. Something like "Each component defines the specified properties it needs in a fragment, and then for any child components it references the fragment exported by that component. It knows that it has children with data dependencies, but it doesn't need to care about the details of that data."

Reply:
Did done. Thanks!